src/templates/ComponentTemplate.jsx

```diff
@@ -1,136 +1,199 @@
 import ComponentProjectForm from "../organisms/ComponentProjectForm";
 import ControlHeader from "../organisms/ControlHeader";
 
 export function ComponentTemplate({
-  component,
-  selectedControl,
   catalogData,
+  catalogVersion,
+  componentData,
+  componentId,
+  projectData,
+  selectedControl,
+  handleSelectControl,
   handleProjectUpdate,
 }) {
-  const { title, description, standard, source } = component.component_data;
+  const { title, description } = componentData;
+  const source = componentData.standards[catalogVersion].source;
+  const componentControls = componentData.standards[catalogVersion].controls;
 
-  const accordionItemsProps = [
-    {
-      title: "CMS Implementation Standards",
-      content: (
-        <p>
-          {selectedControl.implementationStandards ||
-            "No implementation standards found for this control."}
-        </p>
-      ),
-      expanded: false,
-      id: "implementation_standards",
-      headingLevel: "h3",
-    },
-    {
-      title: "CMS Control Guidance",
-      content: (
-        <p>
-          {selectedControl.guidance ||
-            "No control guidance found for this control."}
-        </p>
-      ),
-      expanded: false,
-      id: "control_guidance",
-      headingLevel: "h3",
-    },
-    {
-      title: "Narrative",
-      content: <p>{selectedControl.narrative}</p>,
-      expanded: true,
-      id: "inherited_narratives",
-      headingLevel: "h3",
-    },
-  ];
+  const selectedControlCatalogData = {
+    low: catalogData[catalogVersion].low.controls[selectedControl],
+    moderate: catalogData[catalogVersion].moderate.controls[selectedControl],
+    high: catalogData[catalogVersion].high.controls[selectedControl],
+  };
+  const selectedControlComponentData = componentControls[selectedControl];
+
+  const getImplementationText = (level) => {
+    if (
+      selectedControlCatalogData &&
+      selectedControlCatalogData[level].implementationStandards
+    ) {
+      return selectedControlCatalogData[level].implementationStandards;
+    } else {
+      return "No implementation standards found for this control.";
+    }
+  };
+
+  const getControlGuidanceText = (level) => {
+    if (
+      selectedControlCatalogData &&
+      selectedControlCatalogData[level].guidance
+    ) {
+      return selectedControlCatalogData[level].guidance;
+    } else {
+      return "No control guidance found for this control.";
+    }
+  };
+
+  const getNarrativeText = () => {
+    if (
+      selectedControlComponentData &&
+      selectedControlComponentData.narrative
+    ) {
+      return selectedControlComponentData.narrative;
+    } else {
+      return "No narrative found for this control.";
+    }
+  };
 
   function renderTopSection() {
     return (
       <div className="grid-row">
         <div className="tablet:grid-col padding-top-1">
           <p>
-            <b>Standard:</b> {standard}
+            <b>Standard:</b> {CATALOG_DISPLAY_NAMES[catalogVersion]}
```

Comment:
Thought: I agree with you that this doesn't need a guard clause because we are owning  in the `CATALOG_DISPLAY_NAMES` in our own API